### ISSUE TITLE
Memory allocation of 140015098745856 bytes failed

### ISSUE THREAD

**Issue by patrik-cihal** [qryxip/cargo-equip#192]
I have followed the basic setup. This is my Cargo.toml file: 
```
[package]
name = "love-story"
version = "0.0.0"
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
proconio = { version = "0.4.3", features = ["derive"] }
```
I'm on Ubuntu.

The full error message: 
```
memory allocation of 140015098745856 bytes failed
error: could not bundle the code

- `proconio 0.4.3 (registry+https://github.com/rust-lang/crates.io-index)` as `crate::__cargo_equip::crates::proconio`
- `proconio-derive 0.2.1 (registry+https://github.com/rust-lang/crates.io-index)` as `crate::__cargo_equip::crates::__proconio_derive_0_2_1`


Caused by:
  rust-analyzer error

Caused by:
  server exited
```

**Comment by mizar** [comment]
```toml
[dependencies]
proconio = { version = "0.4.3" }
```

It seems that memory allocation errors are less likely to occur when derive features are not used, but I have no idea of the exact cause at this time.

**Comment by cpchenpi** [comment]
Same issue on Windows. Removing "features = ["derive"]" could solve this problem, but fastout will be unavailable.

**Comment by qryxip** [comment]
I updated the embedded Rust Analyzer to `2023-07-31`, and released [v0.20.0](https://github.com/qryxip/cargo-equip/releases/tag/v0.20.0). Please reopen this issue if the problem remains.

**Comment by patrik-cihal** [comment]
> I updated the embedded Rust Analyzer to `2023-07-31`, and released [v0.20.0](https://github.com/qryxip/cargo-equip/releases/tag/v0.20.0). Please reopen this issue if the problem remains.

For me, the problem is gone. :)
